gobblin/gobblin.sh

```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -euxo pipefail
+
+readonly PACKAGE_URL="gs://gobblin-dist/gobblin-distribution-0.12.0.rc2.tar.gz"
```

Comment:
Any reason to use an RC rather than the latest release version? Also are we maintaining this GCS mirror or is Gobblin?

Reply:
I believe v0.12 will be released any day now, and this is the version I have been toying with.

Gobblin folk are not consistently providing binary builds hence the staged version (owned by us). I was advised to do so by @AngusDavis

Reply:
I added a comment to readme noting that gobblin-dist is owned by Google